Tests/KinematicsDynamics/TestRegressorInverseDynamics.cpp

```diff
@@ -35,25 +39,17 @@ int main() {
     VecX z = 2 * M_PI * VecX::Random(trajPtr->varLength).array() - M_PI;
 
     // Compute inverse dynamics using RegressorInverseDynamics
-    auto start_clock = std::chrono::high_resolution_clock::now();
     regressor_id.compute(z, false);
-    auto stop_clock = std::chrono::high_resolution_clock::now();
-    auto duration = std::chrono::duration_cast<std::chrono::nanoseconds>(stop_clock - start_clock);
-    std::cout << "RegressorInverseDynamics: " << duration.count() << " nanoseconds" << std::endl;
 
     // Compute inverse dynamics using pinocchio::rnea
     trajPtr->compute(z, false);
-    start_clock = std::chrono::high_resolution_clock::now();
     VecX tau_pinocchio = pinocchio::rnea(model, data, trajPtr->q(0), trajPtr->q_d(0), trajPtr->q_dd(0));
-    stop_clock = std::chrono::high_resolution_clock::now();
-    duration = std::chrono::duration_cast<std::chrono::nanoseconds>(stop_clock - start_clock);
-    std::cout << "Pinocchio RNEA: " << duration.count() << " nanoseconds" << std::endl;
-
-    // Compare the results
-    std::cout << "RegressorInverseDynamics result:" << std::endl;
-    std::cout << regressor_id.tau(0).transpose() << std::endl;
-    std::cout << "Pinocchio RNEA result:" << std::endl;
-    std::cout << tau_pinocchio.transpose() << std::endl;
-
-    return 0;
-}
\ No newline at end of file
+
+    // compare the results
+    BOOST_CHECK_SMALL((regressor_id.tau(0) -tau_pinocchio).norm(), 1e-10);
+
```

Comment:
Please avoid adding unnecessary empty lines in your code. Keep things clean and organized.